### PR TITLE
New Apps UI

### LINKED ISSUE #409: New apps UI
https://apps.darwinia.network/

image

### PR DESCRIPTION
**Pull request by nblogist**
Closes https://github.com/darwinia-network/home/issues/409 

This PR once finalized will add a new look to the Darwinia Apps Page.

### CODE REVIEW

**Comment by nblogist** [comment]
PR will be ready after this is answered about the header's behavior: https://github.com/darwinia-network/home/issues/409#issuecomment-2067616087

**Comment by hamidroohi71** [comment]
I did not get any answer about the logic. So I fixed the style according to the prev logic.

**Review by hackfisher — DISMISSED** [review]
LGTM

File: src/components/Hero.tsx [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
+      <div
+        className="inset-0 mt-[3.5rem] h-[189px] w-full bg-cover bg-center lg:absolute lg:mt-0 lg:h-auto lg:opacity-[30%]"
+        style={{
+          backgroundImage: `url(${hero})`,
+        }}
+      ></div>
+      <p className="relative z-10 text-[1.5rem] font-[700] text-white ">
         Explore Dapps building on {"{"}
-        <span className="text-primary">Darwinia</span>
+        <span className="text-primary">Darwinia2</span>
```
Comment (hackfisher) [inline src/components/Hero.tsx]:
```suggestion
        <span className="text-primary">Darwinia</span>
```

**Review by hackfisher — APPROVED** [review]

**Pull request merged**